src/localization.py

```diff
@@ -1,12 +1,20 @@
 import json, string
 
-def get_localized(key):
-    # Find active language
-    with open('Current Language', 'r') as file:
-        language = file.readline().strip('\n')
-    # Find matching key in lanugage json
-    with open(f'Languages/{language}.json', 'r') as file:
-        return json.load(file)[key]
+def get_language(): # Find active language
+    global local_script
+    try:
+        with open('Current Language', 'r') as file:
+            language = file.readline().strip('\n')
+            with open(f"Languages/{language}.json", 'r') as file:
+                pass           
+    except:
```

Comment:
Idea with #59 was to cover case when requested language doesn't have key. If I'm not wrong, that is not covered by this, right?
Looks like this covers case when language file doesn't exist at all.